sdk/formrecognizer/ai-form-recognizer/test/public/formtrainingclient.spec.ts

```diff
@@ -241,7 +241,6 @@ matrix([[true, false]] as const, async (useAad) => {
        */
       describe("account properties", () => {
         it("has trained models and limits", async () => {
-          const trainingClient = new FormTrainingClient(endpoint(), makeCredential(useAad));
           const properties = await trainingClient.getAccountProperties();
```

Comment:
@willmtemple I think it was a mistake to redefine `trainingClient` here, no?

Reply:
Yeah, I think I probably lifted the earlier declaration of training client up a level and then left this in by accident. It should be harmless to remove if you want to yank it. Otherwise, it should also be harmless to leave it in and I can get rid of it in a different PR unless your linter is blocking on this.

Reply:
Great! I will keep my change then.